generator/src/main/java/com/graphicsfuzz/generator/tool/CustomMutatorSever.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2018 The GraphicsFuzz Project Authors
```

Comment:
@paulthomson do you have thoughts about whether generator/src/main/java/com/graphicsfuzz/generator/tool/ is a suitable location for this class?  Would it perhaps be better for us to add a new Maven module for libFuzzer integration?

Reply:
I feel it is probably OK here at this stage.